src/aind_metadata_service/server.py

```diff
@@ -3,11 +3,13 @@
 import os
 
 from fastapi import FastAPI
+from fastapi.responses import RedirectResponse
 
 from aind_metadata_service.labtracks.client import LabTracksClient
 from aind_metadata_service.sharepoint.client import SharePointClient
 
 app = FastAPI()
+favicon_path = "https://alleninstitute.org/static/images/favicons/alleninstitute/favicon.png"  # noqa: E501
```

Comment:
Don't hardcode endpoints. You can assume it's set in an env var.

Reply:
Where do we set the variable?

Reply:
set this to load from an env variable FAVICON_PATH